README.md

```diff
@@ -110,42 +110,15 @@ kubectl delete cluster "${CLUSTER_NAME:-"test1"}" -n metal3
 
 ### Deploying with Tilt
 
-It is possible to use Tilt to run the CAPI and CAPM3 components. For this, run:
+It is possible to use Tilt to run the BMO and CAPM3 components. For this, run:
 
 ```sh
 export EPHEMERAL_CLUSTER="tilt"
+export IRONIC_TLS_SETUP=false
+export IRONIC_BASIC_AUTH=false
 make
 ```
 
-Then clone the Cluster API Provider Metal3 repository, and follow the
-[instructions](https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/docs/dev-setup.md#tilt-development-environment).
-That will mostly be the three following blocks of commands.
-
-```sh
-source lib/common.sh
-source lib/network.sh
-source lib/images.sh
-```
-
-and go to the CAPM3 repository and run
-
-```sh
-make tilt-settings
-```
-
-Please refer to the CAPM3 instructions to include BMO and IPAM. Then run :
-
-```sh
-make tilt-up
-```
-
-Once the cluster is running, you can create the BareMetalHosts :
-
-```sh
-kubectl create namespace metal3
-kubectl apply -n metal3 -f /opt/metal3-dev-env/bmhosts_crs.yaml
-```
-
```

Comment:
The first one is part of 03 script
`kubectl create namespace metal3
`
The second one is part of 03. It is also present in capm3 Makefile. Therefore, no need to be run manually.
`kubectl apply -n metal3 -f /opt/metal3-dev-env/bmhosts_crs.yaml
`

Reply:
but we should leave cloning capm3 repo, make tilt-settings, make tilt-up notes at least right, otherwise how the user will be able to deploy the cluster with these given instructions?

Reply:
Thanks. 
re-added the removed sections